cloud/jenkins/pxc_operator_minikube.groovy

```diff
@@ -267,7 +267,9 @@ pipeline {
                     runTest('affinity')
                     runTest('one-pod')
                     runTest('self-healing-advanced')
+                    runTest('self-healing-advanced-chaos')
                     runTest('operator-self-healing')
+                    runTest('operator-self-healing-chaos')
```

Comment:
no self-healing-advanced-chaos ?

Reply:
`self-healing-advanced-chaos` is added above